### ISSUE TITLE
Exposing accessibility prop for access keys

### ISSUE THREAD

**Issue by rurikoaraki** [microsoft/react-native-windows#10353]
### Summary

Add a prop to set the access key UIA property on a component.

### Motivation

RN Win32 is working on a feature which requires exposing a prop to add access key accessibility information to a component. Guidance from https://github.com/microsoft/react-native-windows/issues/3450 mentions using accessibilityHint to set the property, but we've found that accessibilityHint doesn't consistently get announced by Narrator for certain control types like MenuItems on win32. So, we will be going with exposing a prop to set the UIA AccessKey property.

Opening this issue to revisit this guidance for RNW and discuss if proper support for access key accessibility should be added. It would be good to have consistency across platforms for exposing accessibility information.

### Basic Example

<View onKeyDown={handleAccessKey()} accessibilityAccessKey={'a'}>
    <Text>Test</Text>
</View>

**Comment by chrisglein** [comment]
For XAML this would likely map to [AccessKey](https://docs.microsoft.com/en-us/uwp/api/windows.ui.xaml.uielement.accesskey?view=winrt-22621)

**Comment by acoates-ms** [comment]
This should be implemented as [aria-keyshortcuts](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-keyshortcuts)?
